packages/node/src/integrations/requestdata.ts

```diff
@@ -61,10 +64,15 @@ export class RequestData implements Integration {
    */
   public name: string = RequestData.id;
 
+  // TODO(v8): Remove this method as it will become redundant once we strictly type the request interface on `sdkProcessingMetadata`.
   /**
    * Function for adding request data to event. Defaults to `addRequestDataToEvent` from `@sentry/node` for now, but
    * left as a property so this integration can be moved to `@sentry/core` as a base class in case we decide to use
    * something similar in browser-based SDKs in the future.
+   *
+   * @deprecated In the future the transaction API will likely have a more strictly typed interface, meaning
```

Comment:
Do we need to deprecate a protected method? 🤔 do we consider this public API at all?

Reply:
Yes we consider protected methods as part of public API. This deprecation notice is probably not 100% necessary because I doubt anybody uses this, however I also didn't wanna give anybody bad surprises.

Reply:
I'd say that is pretty internal/intimate - it is underscored, marked as protected in TS and thus will not show up anywhere? We have _a lot_ of private methods and I don't think we can/should consider all of these public API. (protected is private except it can be sub-classed, which I'd say is not too relevant for an integration like this?)

So to be clear I don't care at all about adding this here, that's all good IMHO 👍 just wanting to clarify also for other things 😅